package.json

```diff
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "http://github.com/auth0/mongo-heartbeat.git"
   },
-  "author": "José F. Romaniello <jfromaniello@gmail.com> (http://joseoncode.com)",
```

Comment:
even if the repository is on the auth0 organization I am the author and the person people should contact in case of troubles. Please do not change this line.